src/business-logic/services/test/execute-action-service.spec.ts

```diff
@@ -254,9 +254,9 @@ describe(ExecuteActionService.name, () => {
 
           const action: PieceAction = createPieceAction(PieceActionType.REPLACE_PIECE)
 
-          const mutateActionMethods: MutateActionWithPieceMethods = {
+          const mutateActionMethods: MutateActionWithOnAirAndNextPiecesMethods = {
             type: MutateActionType.PIECE,
-            updateActionWithPiece: (action) => action,
+            updateActionWithPiece: (action: Action, _onAirPiece: Piece | undefined, _nextPiece: Piece | undefined) => action,
```

Comment:
```suggestion
            updateActionWithPiece: (action: Action, _onAirPiece?: Piece, _nextPiece?: Piece) => action,
```
Again, this is just a suggestion :)

Reply:
Updated